packages/http-signature-utils/src/utils/signatures.ts

```diff
@@ -36,15 +28,19 @@ export const createSignatureHeaders = async ({
   if (request.body) {
     components.push('content-digest', 'content-length', 'content-type')
   }
-  const { headers } = await httpsig.sign(request, {
-    components,
-    parameters: {
-      created: Math.floor(Date.now() / 1000)
+
+  const signingKey = createSigner(privateKey, 'ed25519', keyId)
+
+  const { headers } = await httpbis.signMessage(
+    {
+      key: signingKey,
+      name: 'sig1',
```

Comment:
In the current version, the default name for the signature is `sig`.

Reply:
Is it me or is there isn't any docs re: `name` and `params` attributes?

Reply:
Documentation for the name and params attributes is missing. By looking at the type definition of `signMessage` and checking the first argument type, we can identify all the configurable options available:
```ts
export interface SignConfig extends CommonConfig {
    key: SigningKey;
    /**
     * The name to try to use for the signature
     * Default: 'sig'
     */
    name?: string;
    /**
     * The parameters to add to the signature
     * Default: see defaultParams
     */
    params?: string[];
    /**
     * The HTTP fields / derived component names to sign
     * Default: none
     */
    fields?: string[];
    /**
     * Specified parameter values to use (eg: created time, expires time, etc)
     * This can be used by consumers to override the default expiration time or explicitly opt-out
     * of adding creation time (by setting `created: null`)
     */
    paramValues?: SignatureParameters;
    /**
     * A list of supported algorithms
     */
    algs?: Algorithm[];
}
```